.github/workflows/release.dsg.hotfix.yml

```diff
@@ -0,0 +1,20 @@
+name: Release DSG Hotfix
+run-name: Release DSG hotfix (sha:${{ github.sha }})
+on:
```

Comment:
The job trigger is on a new-line in all of the other workflows, lets keep it that way for readability

Reply:
```suggestion

on:
```